tests/accelerators/test_accelerator_connector.py

```diff
@@ -326,26 +340,25 @@ def on_fit_start(self, trainer, pl_module):
 @RunIf(special=True)
 def test_accelerator_choice_ddp_cpu_and_plugin(tmpdir):
     """Test that accelerator="ddp_cpu" can work together with an instance of DDPPlugin."""
-    _test_accelerator_choice_ddp_cpu_and_plugin(tmpdir, ddp_plugin_class=DDPPlugin)
+    _test_accelerator_choice_ddp_cpu_and_plugin(tmpdir, ddp_strategy_class=DDPPlugin)
```

Comment:
```suggestion
    _test_accelerator_choice_ddp_cpu_and_strategy(tmpdir, ddp_strategy_class=DDPPlugin)
```
For other changes as well

Reply:
Done